regen/warnings.pl

```diff
@@ -1102,6 +1126,11 @@ =head2 "Negative warnings"
     no warnings  'experimental';
     use warnings 'experimental::somefeature';
 
+As experimental features become regular features of Perl,
+the corresponding warnings are not thrown anymore.
+For backwards compatibility, it is still possible to disable or enable
```

Comment:
Instead of:

> It is still possible to request turning on or off these warnings, but doing so does exactly nothing.

how about:

> It is still possible to request turning on or off these warnings, but doing so has no effect whatsoever.

The avoidance of repetition might be a French stylistic thing, though.